cmd/internal/cli/sign.go

```diff
@@ -87,7 +87,6 @@ var SignCmd = &cobra.Command{
 	Args:                  cobra.ExactArgs(1),
 
 	Run: func(cmd *cobra.Command, args []string) {
-
 		// args[0] contains image path
 		fmt.Printf("Signing image: %s\n", args[0])
 		if err := doSignCmd(args[0], keyServerURI); err != nil {
```

Comment:
while we are, can we remove `keyServerURI` which is global and maybe replace `sylog.Errorf/os.Exit` by `sylog.Fatalf()` unless there is need to return a specific exit code ?

Reply:
Also removed the error return, to keep it consistent with `verify.go`.